conformance/utils/kubernetes/helpers.go

```diff
@@ -386,9 +495,14 @@ func HTTPRouteMustHaveCondition(t *testing.T, client client.Client, timeoutConfi
 		}
 
 		parents := route.Status.Parents
-
 		var conditionFound bool
 		for _, parent := range parents {
+
+			if err := ConditionsHaveLatestObservedGeneration(route, parent.Conditions); err != nil {
+				t.Logf("HTTPRoute(controller=%v,ref=%#v) %v", parent.ControllerName, parent, err)
```

Comment:
when looking at the test logs, these lines end up being hard to read, e.g.

```
=== RUN   TestConformance/HTTPRouteObservedGenerationBump/observedGeneration_should_increment
    helpers.go:209: Gateways and Pods in gateway-conformance-infra namespaces ready
    helpers.go:502: HTTPRoute(controller=projectcontour.io/gateway-controller,ref=v1beta1.RouteParentStatus{ParentRef:v1beta1.ParentReference{Group:(*v1beta1.Group)(0xc00041e310), Kind:(*v1beta1.Kind)(0xc00041e350), Namespace:(*v1beta1.Na
mespace)(nil), Name:"same-namespace", SectionName:(*v1beta1.SectionName)(nil), Port:(*v1beta1.PortNumber)(nil)}, ControllerName:"projectcontour.io/gateway-controller", Conditions:[]v1.Condition{v1.Condition{Type:"Accepted", Status:"True",
 ObservedGeneration:1, LastTransitionTime:time.Date(2022, time.December, 16, 17, 46, 3, 0, time.Local), Reason:"Accepted", Message:"Accepted HTTPRoute"}}}) expected observedGeneration to be updated for all conditions, only 0/1 were update
d. stale conditions are: Accepted
```

if we want to keep these logs might be good to pare this down a bit (and in other places the same pattern is used)

Reply:
Done - just using `namespace/name` of the parent ref